flutter_vlc_player/lib/vlc_player_flutter.dart

```diff
@@ -0,0 +1,37 @@
+library vlc_player_flutter;
+
+import 'dart:async';
+import 'dart:io';
+import 'dart:typed_data';
+import 'dart:ui';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+import 'package:flutter_vlc_player_platform_interface/vlc_player_flutter_platform_interface.dart';
+
+export 'package:flutter_vlc_player_platform_interface/vlc_player_flutter_platform_interface.dart'
+    show
+        HwAcc,
+        VlcMediaEvent,
+        VlcMediaEventType,
+        VlcRendererEvent,
+        VlcRendererEventType,
+        DataSourceType,
+        VlcPlayerOptions,
+        VlcAdvancedOptions,
+        VlcAudioOptions,
+        VlcRtpOptions,
+        VlcStreamOutputOptions,
+        VlcVideoOptions;
+
+part 'src/enums/playing_state.dart';
+
+part 'src/vlc_player_controller.dart';
+
+part 'src/flutter_vlc_player.dart';
+
+part 'src/vlc_app_life_cycle_observer.dart';
+
+part 'src/vlc_player_value.dart';
```

Comment:
Have you considered avoiding `part` and `part of` directives? They tend to make code navigation a bit more messy than with only `import`'s. 

Also usage of `part` and `part of` is discouraged by the Dart team.
https://dart.dev/guides/libraries/create-library-packages#organizing-a-library-package

Reply:
Yea this shouldnt be a big deal to resolve, part of why we need code reviews! We forget about minute things when deep in the code.

Reply:
fixed